### DISCUSSION TITLE
How can I login on self-hosted illa-builder?

### DISCUSSION THREAD

**Discussion by exitia-com2us** [illacloud/illa-builder#672] [Q&A]
I tried to login with `root` / `password` as README.txt says, but it makes an error(wrong email format).

And there is `/user/register` path to register but no `register` button or link on main page. Also, sending access code to email always fails.

How can I login or register properly on self-hosted?

* I used [docker-compose](https://github.com/illacloud/deploy-illa-manually/blob/main/docker-compose/docker-compose-with-official-images.yml) locally to run illa-builder.

**Comment by karminski** [comment]
@exitia-com2us hi, the default login info is only for the "self-host“ version. And the docker-compose / Kubernetes deployments are "cloud" versions for production only. 

To deploy the "self-host" version please follow the:

- illa cli deploy tools: https://github.com/illacloud/illa
- deploy all-in-one illa image with docker: https://github.com/illacloud/deploy-illa-manually/tree/main/docker

To deploy the "cloud" version please follow the:

- docker-compose: https://github.com/illacloud/deploy-illa-manually/tree/main/docker-compose
- Kubernetes: https://github.com/illacloud/deploy-illa-manually/tree/main/kubernetes

please note that all of these deploy methods require the env param, so please deploy with the cli tools or manual deploy scripts.
